app/main.py

```diff
@@ -1,3 +1,15 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable, Any
+
+
+def cache(func: tuple) -> Callable:
```

Comment:
Also one addition here, func that is passed as an argument to `cache` should be not `tuple` but `Callable` as well. Pay attention to typing for incoming arguments as well.